### ISSUE TITLE
Contributors block

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#1005]
Cross content block for the sidebar showing each contributor's avatar and name, and links to their .org profile.

It should look similar to this from Tutorials, but without the description, and in a vertical stack so that multiple contributors can be listed:

![Image](https://user-images.githubusercontent.com/1017872/194983253-67ea0fc8-9fb2-4e8d-bfec-67aea67f0ead.jpg)

It will receive an array of contributors, but this list could be sourced from different data fields depending on the post type:

| Post type | Proposed field |
|-|-|
| Tutorial | Other Contributors (add presenter to this field) |
| Course | Teacher |
| Online Workshop | N/A |
| Lesson Plan | Needs 'Contributors' field added |

**Comment by hlashbrooke** [comment]
I think this update should also standardise the field used for contributors, so each post type just has one field, and it's the same for all the post types. The best bet would be to use one of the existing fields that tutorials have, remove the other one, and then apply it to the other post types as well.

**Comment by kaitohm** [comment]
@adamwoodnz Flagging this as another feature that would be great to have implemented with the redesign.

With us streamlining content types, all we'd be looking at now is adding a Contributor field to Sensei Lessons. This should allow for one or more contributors, and link to their WordPress.org profile on the front end.

**Comment by adamwoodnz** [comment]
> @adamwoodnz Flagging this as another feature that would be great to have implemented with the redesign.
> 
> With us streamlining content types, all we'd be looking at now is adding a Contributor field to Sensei Lessons. This should allow for one or more contributors, and link to their WordPress.org profile on the front end.

Added to the project, we'll see if we can get to it

**Comment by westnz** [comment]
Can we confirm that this block is only for direct content creators and not reviewers as well?

**Comment by kathrynwp** [comment]
Discussed with @westnz  the idea of including lesson reviewers as part of a Contributors block, perhaps names only, as a course will have multiple reviewers and we don't want this block taking up too much space. 

Also note that we no longer need this on lesson plans or tutorials, as those content types are being deprecated.

Would be good to see a design for this.

### Suggestion

Presenter: Gravatar / name (linked to WP.org profile)
Editor: Gravatar / name (linked to WP.org profile)
Script: Gravatar / name (linked to WP.org profile)
Review: name (linked to WP.org profile) - comma-separated list of reviewers

@westnz What do you think of this idea? The Contributor block is only meant for the course overview page, not individual lesson pages, right? Are there other roles besides the ones I suggested? Should we separate them out like this?

**Comment by westnz** [comment]
Hey Kathryn

Ben mentioned:
> With us streamlining content types, all we'd be looking at now is adding a Contributor field to Sensei Lessons

But I realise we don't want all of this information displayed on each lesson as a person progresses through a learning pathway. Having it on the course or learning pathway overview page makes more sense to me.

> Suggestion
> Presenter: Gravatar / name (linked to WP.org profile)
> Editor: Gravatar / name (linked to WP.org profile)
> Script: Gravatar / name (linked to WP.org profile)
> Review: name (linked to WP.org profile) - comma-separated list of reviewers

I like this suggestion, but for some lessons, Jonathan, Cynthia, or I would've been the scriptwriter, editor, and presenter. Therefore, it might be better to use a general term such as 'Content Creators' and list the names of those involved.  Agreed that the reviewers could be separate and just have their names displayed.

**Comment by adamwoodnz** [comment]
@WordPress/meta-design I think this really needs design input please.

**Comment by jasmussen** [comment]
Hey, happy to take a look. Perhaps I'm not deep enough into the inner flows, but it's a little hard to know exactly what to suggest here. Can you provide some links or some more screenshots to the missing features or problematic aspects you'd like input on? I understand this is a list of presenters with some metadata, does such a list exist today? On which page should it live?

**Comment by adamwoodnz** [comment]
In essence we need a way of displaying a list of content creators on a course view, with or without avatars, with each linked to their .org profile. Potentially a second list of reviewers which could be visually different and less prominent.

There is nothing existing that I know of, other than the original screenshot from the old theme in the description.

I'm not sure of the exact metadata yet, but safe to say we'll have first and last name, .org username and we can fetch an avatar if necessary.

**Comment by jasmussen** [comment]
Would this pattern from a work-in-progress Translate refresh work?

![Screenshot 2024-08-30 at 08 58 14](https://github.com/user-attachments/assets/326c4ce1-4721-4003-ab5d-127275d46135)

[Figma](https://www.figma.com/design/Q3RpUkAJhTaKVmXdGGJIBs/Translating-WordPress?node-id=41-951&t=WVjBQdLafz6wt5qD-1).

**Comment by adamwoodnz** [comment]
> Would this pattern from a work-in-progress Translate refresh work?

Yep, although I wonder whether it should be across the bottom of the course in a wide format like this, or stacked at the bottom of the sidebar. Seems like meta data, and that is generally in the sidebar.

**Comment by jennydupuy** [comment]
Hi,

We could also add a mention of the contributors who translated the content, as it's a pretty tough job to do.
In the french translation of documentation, we have added the name of translators and reviewers. See for example: https://fr.wordpress.org/support/article/navigation-block/

We also need to anticipate updates that may be made by other contributors.

I hope what I'm proposing makes sense to you too.

**Comment by ironnysh** [comment]
Hi, hope it’s not too late to chime in.

I agree with @westnz’s [comment](https://github.com/WordPress/Learn/issues/1005#issuecomment-2305874287) above:
> But I realise we don't want all of this information displayed on each lesson as a person progresses through a learning pathway. Having it on the course or learning pathway overview page makes more sense to me.

I'd like to suggest these two alternatives:

### Option 1

Add a _**Credits**_ page to each course and link it from the sidebar. Something like this:
<img width="790" alt="credits" src="https://github.com/user-attachments/assets/7655fcd6-95e4-45ac-b74d-966725008778">

The page itself will either use the standard template, or a predefined pattern with a table/list of roles and names and relevant metadata (see @kathrynwp's suggestion above):
> Presenter: Gravatar / name (linked to WP.org profile)
Editor: Gravatar / name (linked to WP.org profile)
Script: Gravatar / name (linked to WP.org profile)
Review: name (linked to WP.org profile) - comma-separated list of reviewers 

The same is done in the [WordPress Developer Resources](https://developer.wordpress.org/themes/credits/) (although less “designed” 😉).

### Option 2

Add a _**Credits**_ element at the bottom of the table of contents accordion.
